templates/registry/markdown/enum_macros.j2

```diff
@@ -1,10 +1,11 @@
 {% import 'stability.j2' as stability %}
+{% macro filter(member) %}{% if snippet_type is not defined or (member.deprecated is none or member.deprecated == "") %}{{ "True" }}{% else %}{{ "False" }}{% endif %}{% endmacro %}
```

Comment:
nit: I might be late to the review, but filter usually filters things out, so I was stumbled upon usage, where we `filter` member but still continue to process it later
So semantically it should be `False` for good member and `True` for deprecated one.

Reply:
```suggestion
{% macro filter_deprecated(member) %}{% if snippet_type is not defined or (member.deprecated is none or member.deprecated == "") %}{{ "False" }}{% else %}{{ "True" }}{% endif %}{% endmacro %}
```

something like this